### DISCUSSION TITLE
What is the difference between preset targets and browserslistrc

### DISCUSSION THREAD

**Discussion by yoyo837** [babel/babel#14972] [Q&A] [answered]
How to unify into a single configuration?

**Comment by nicolo-ribaudo** [comment, marked as answer]
They are the same: you can either put your targets in the `targets` option or in `.browserslistrc`. `.browserslistrc` has the advantage that it's also read by other tools (for example, PostCSS), but it has the disadvantage that it's one additional configuration file.

**Reply by yoyo837** [reply]
Thanks for your reply. Who takes precedence, if they both exist?

**Reply by nicolo-ribaudo** [reply]
The targets option.
